src/components/main/main-navbar.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import arrow from '../../images/main navbar/arrow.png';
+
+const MainNavbar = () => {
+  return (
+    <div className="main-navbar">
+      <div clas="main-navbar-open-aside">
+        {/* <span style="fontSize:30px;cursor:pointer" onClick="openNav()">&#9776;</span>         */}
+      </div>
+      <div className="search-box">
+        <input className="search-txt" type="text" placeholder="Search..." />
```

Comment:
where is the realization of search logic?

Reply:
not fixed

Reply:
I make small search logic, very primitive.